dvc/command/repro.py

```diff
@@ -35,6 +35,10 @@ def run(self):
                     metrics = self.repo.metrics.show()
                     logger.info(_show_metrics(metrics))
 
+                logger.info(
+                    "To save your updates to remote storage, run 'dvc push'."
```

Comment:
Maybe something like "Use 'dvc push' to send your updates to remote storage."?

Reply:
No problem, done!